Modules/AloeExpress.xojo_code

```diff
@@ -178,21 +178,44 @@ Protected Module AloeExpress
 		  // Logs an event.
 		  // See LogLevel enumeration for log levels.
 		  
-		  
-		  Select Case CType( Level, Integer )
```

Comment:
Using CType conversion was not necessary and theoretically using CType to convert between integers and enums could introduce bugs if an integer that is not a valid enum value is passed.